### ISSUE TITLE
Hot Edge Required Pressure doesn't work when Dash-to-Panel intellihide Required Pressure is set

### ISSUE THREAD

**Issue by Malix-off** [jdoda/hotedge#34]
Hot Edge Required Pressure doesn't work when Dash-to-Panel intellihide Required Pressure is set

## Hot Edge - Activation Pressure

![image](https://github.com/jdoda/hotedge/assets/76160668/a38c7eee-f32e-4f6b-98b6-060e66e9d051)

## Dash to Panel - Intellihide - Required Pressure

![image](https://github.com/jdoda/hotedge/assets/76160668/2a29e51e-e52b-487a-a42d-e82b92ca0397)

**Comment by jdoda** [comment]
Yeah, what happens probably depends both on the settings and the order the extensions are loaded in, which isn't necessarily consistent. I don't think having two different pressure barriers sharing the same space is a supported use case, unfortunately, so I don't see much chance of getting this behaving sensibly. Out of curiosity though, how do you expect this to work? I guess you want the panel to appear with a little bit of pressure and then the overview to activate if you apply more pressure, or do you want some other behaviour?

**Comment by Malix-off** [comment]
Yeah, I guess you're right

One way would be to make an explicit communication between the other extension, though

I expected one of the two behaviours to be possible:
1. The panel to appear with a little bit of pressure and then the overview to activate if you apply more pressure
2. The panel to only show in overview (never show in desktop), and the overview to require some hot edge pressure to open
